app/experimenter/nimbus-ui/src/components/ExperimentNotFound/index.tsx

```diff
@@ -0,0 +1,26 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { Link } from "@reach/router";
+import React from "react";
+import { BASE_PATH } from "../../lib/constants";
+import AppLayout from "../AppLayout";
+
+const ExperimentNotFound = ({ slug }: { slug: string }) => (
```

Comment:
Went a little beyond what the ticket asks for here but dig the results:

<img width="1181" alt="image" src="https://user-images.githubusercontent.com/13018240/97236237-a8e30580-17b2-11eb-859d-05c4b02ac0be.png">

Reply:
I like it